presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -0,0 +1,197 @@
+=================
+Iceberg Connector
+=================
+
+Overview
+--------
+
+Apache Iceberg is an open table format for huge analytic datasets.
+The Iceberg connector allows querying data stored in
+files written in Iceberg format, as defined in the
+`Iceberg Table Spec <https://iceberg.apache.org/spec/>`_.
+
+The Iceberg connector supports:
```

Comment:
I think we should remove this list, as we don't have it for other connectors. Many of these are basics that are expected, and can be or are documented implicitly with examples.

Reply:
Removed.